airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/BasicAcceptanceTests.java

```diff
@@ -953,6 +1125,7 @@ public void testIncrementalSyncMultipleStreams() throws Exception {
   // This test is disabled because it takes a couple of minutes to run, as it is testing timeouts.
   // It should be re-enabled when the @SlowIntegrationTest can be applied to it.
   // See relevant issue: https://github.com/airbytehq/airbyte/issues/8397
+  @Test
```

Comment:
Do you want to re-enable this test?

Reply:
No, the `@Disabled` is still below. I expected the `@Test` to have been removed as part as a not so great merge conflict resolve.